free-gpus

```diff
@@ -1,9 +1,38 @@
 #!/bin/bash
+
+print_usage () {
+  cat << EOM
+Usage: $0
+    List nodes with free GPUs, as well as a breakdown of how many GPUs in each node are available.
+
+Arguments:
+    None.
+
+Output:
+    Lists each node with at least one free GPU on separate lines.
+    Also lists a breakdown of the states of the GPUs:
+        -- The 'in_use' column shows the number of GPUs on the node that are currently allocated to a job.
```

Comment:
You know... at some point we should change this command to actually return the number of free gpus there are available! Currently you have to do usable - in_use in your head (not a big ask, but doesn't make it entirely clear).